web/src/routes.js

```diff
@@ -1,30 +1,23 @@
 import NoMatch from './components/NoMatch';
 import CreateAccount from './containers/admin/CreateAccount';
+import Dashboard from './containers/Dashboard';
 import ApdApplication from './containers/ApdApplication';
-import AdminDashboard from './containers/admin/AdminDashboard';
 import EditAccount from './containers/admin/EditAccount';
 import MyAccount from './containers/admin/MyAccount';
 import Login from './containers/Login';
 import Logout from './containers/Logout';
 
-import StateDash from './containers/StateDashboard';
-
 const routes = [
-  { path: '/', component: StateDash, exact: true, nonPrivate: false },
-  { path: '/apd', component: ApdApplication, exact: true, nonPrivate: false },
-  { path: '/login', component: Login, nonPrivate: true },
-  { path: '/logout', component: Logout, nonPrivate: false },
+  { path: '/', component: Dashboard, exact: true, isPublic: false },
```

Comment:
Changed `nonPrivate` to `isPublic` because that seems easier to read to me.  I'll revert if anyone disagrees, but the negation there always threw me off.  "Yes, it is non-private" vs "Yes, it is public."

Reply:
Thank you!! This really confused me the first time I saw it. ^_^